src/stores/AppStore.js

```diff
@@ -734,9 +737,11 @@ export default types
       }
     }
 
-    function prevTask() {
-      if (self.canGoPrevTask) {
-        const { taskId, annotationId } = self.taskHistory[self.taskHistory.findIndex((x) => x.taskId === self.task.id) - 1];
+    function prevTask(e, shouldGoBack = false) {
+      const length = !shouldGoBack ? self.taskHistory.findIndex((x) => x.taskId === self.task.id) - 1 : self.taskHistory.length - 1;
```

Comment:
```suggestion
      const length = shouldGoBack ? self.taskHistory.length - 1 : self.taskHistory.findIndex((x) => x.taskId === self.task.id) - 1;
```
Reorganizing this in the affirmative is a bit easier to read.